content/docs/rfcs/17/README.md

```diff
@@ -153,28 +162,37 @@ message WakuMessage {
 ## RateLimitProof
 
 The `proof` field is an array of 256 bytes and carries the zkSNARK proof as explained in the [Publishing process](##Publishing).
-The proof asserts that:
-1. The message publisher is the current member of the group i.e., her/his identity commitment key is part of the membership group Merkle tree with the root `merkle_root`.
-2. `share_x` and `share_y`  are correctly computed.
-3. The `nullifier` is constructed correctly.
 
-Other fields of the `RateLimitProof` message are the public inputs to the rln circuit and used for the generation of the `proof`.
+Other fields of the `RateLimitProof` message are the public inputs to the [rln circuit](/spec/32) and used for the generation of the `proof`.
 
-The `merkle_root` is an array of 32 bytes which holds the root of membership group Merkle tree at the time of publishing the message.
+The `merkle_root` is an array of 32 bytes in little-endian order which holds the root of membership group Merkle tree at the time of publishing the message.
```

Comment:
These data types can we put them in a more clear list, table or ABNF form if protobuf isn't precise enough? E.g. https://rfc.vac.dev/spec/7/#abnf

Reply:
I moved them inside a table https://github.com/vacp2p/rfc/pull/484/commits/5eecaf2a214ec384e40669b306e7179984439956